Objective-C/CBLDatabase.mm

```diff
@@ -1365,23 +1155,24 @@ - (bool) resolveConflictInDocument: (NSString*)docID
         // Get latest local and remote document revisions from DB
         CBL_LOCK(self) {
             // Read local document:
-            localDoc = [[CBLDocument alloc] initWithDatabase: self
-                                                  documentID: docID
-                                              includeDeleted: YES
-                                                contentLevel: kDocGetCurrentRev
-                                                       error: outError];
+            CBLCollection* c = [self defaultCollectionOrThrow];
```

Comment:
This `- (bool)resolveConflictInDocument:withConflictResolver:error` should be moved to the collection.